pkgs/applications/science/math/getdp/default.nix

```diff
@@ -1,14 +1,15 @@
 { lib, stdenv, fetchurl, cmake, gfortran, blas, lapack, mpi, petsc, python3 }:
 
-stdenv.mkDerivation rec {
+let
+  mpiSupport = petsc.passthru.mpiSupport;
+in stdenv.mkDerivation rec {
```

Comment:
Not sure I see the point of this change.

Reply:
`mpiSupport` was moved to `passthru` in `petsc`, so fixed the path. To not pollute the derivation attrset also made it a variable not an attribute.

Reply:
Should `mpiSupport` be overrideable from `getdp`? If so, it should stay an attribute and be referred to using the `finalAttrs` pattern.

Reply:
No, `mpiSupport` should normally be tied to the respective flag in PETSc. Having PETSc with MPI (the default in nixpkgs) and `mpiSupport = false` in getDP leads to build error.